Tests/Marketplace/marketplace_services.py

```diff
@@ -3875,6 +3885,21 @@ def store_successful_and_failed_packs_in_ci_artifacts(packs_results_file_path: s
         packs_results[stage].update(successful_packs_dict)
         logging.debug(f"Successful packs {successful_packs_dict}")
 
+    if successful_uploaded_dependencies_zip_packs:
+        successful_uploaded_dependencies_zip_packs_dict = {
+            BucketUploadFlow.SUCCESSFUL_UPLOADED_DEPENDENCIES_ZIP_PACKS: {
+                pack.name: {
+                    BucketUploadFlow.STATUS: pack.status,
+                    BucketUploadFlow.AGGREGATED: pack.aggregation_str if pack.aggregated and pack.aggregation_str
```

Comment:
Don't think we need the aggregated key here

Reply:
Fixed